projects/packages/backup/src/js/selectors/test/api.js

```diff
@@ -0,0 +1,93 @@
+import selectors from '../api';
+
+describe( 'apiSelectors', () => {
+	describe( 'getAPIRoot()', () => {
+		it( 'should return null when API state is an empty object', () => {
+			const state = {
+				API: {},
+			};
+			const output = selectors.getAPIRoot( state );
+			expect( output ).toBeNull();
+		} );
+
+		it( 'should return null when API state does not include WP_API_root', () => {
+			const state = {
+				API: {
+					dummyItem: 'dummyValue',
+				},
+			};
+			const output = selectors.getAPIRoot( state );
+			expect( output ).toBeNull();
+		} );
+
+		it( 'should return WP_API_root value if API object includes it', () => {
+			const state = {
+				API: {
+					WP_API_root: 'https://wordpress.com/wp-json/',
+				},
+			};
+			const output = selectors.getAPIRoot( state );
+			expect( output ).toBe( 'https://wordpress.com/wp-json/' );
+		} );
+	} );
```

Comment:
Nit-picking here. `it.each` (aka data provider) can be helpful here to avoid duplicated code for each test case.

Reply:
Thank you! I just [refactored](https://github.com/Automattic/jetpack/pull/28130/files#diff-119d49fa76e569468e8ad2aa7c221af8ce8b55d61ef6f552f3fd709367363b57) this test to avoid duplicated code. Could you take another [look](https://github.com/Automattic/jetpack/pull/28130/files#diff-119d49fa76e569468e8ad2aa7c221af8ce8b55d61ef6f552f3fd709367363b57)?